### PR TITLE
enh(installation): improve topic on installing from a VM

### PR DESCRIPTION
**Pull request by cg-tw**
## Description
Improved topic on installing from a VM.

## Target serie

- [ ] 20.04.x
- [ ] 20.10.x
- [x] 21.04.x
- [x] 21.10.x (master)

### CODE REVIEW

**Review by tuntoja — COMMENTED** [review]
It may be a good idea to add that default locale is en_GB.UTF-8 and keyboard layout is gb so that user knows he should use qwerty keys to type 'ip addr'

**Review by mdebauchez — APPROVED** [review]

File: fr/installation/installation-of-a-central-server/using-virtual-machines.md [hunk 1]
```diff
@@ -1,106 +1,207 @@
...
 
-> La machine virtuelle ne contient pas de configuration de carte réseau. Il sera
-> nécessaire d'ajouter une carte réseau avant de démarrer le serveur.
+    • Si vous souhaitez être contacté par Centreon, entrez vos informations de contact, puis
+cliquez sur **Download**.
 
-## Import
+    • Dans le cas contraire, cliquez sur **Direct download**.
+
+5. Le fichier téléchargé est une archive zippée : extrayez son contenu dans le répertoire désiré.
```
Comment (tuntoja) [inline fr/installation/installation-of-a-central-server/using-virtual-machines.md:40]:
```suggestion
5. Le fichier téléchargé est une archive compressée: extrayez son contenu dans le répertoire désiré.
```

File: en/installation/installation-of-a-central-server/using-virtual-machines.md [hunk 2]
```diff
@@ -1,93 +1,187 @@
...
-
-<!--Oracle Virtualbox-->
-\
-Extract the contents of the archive and double-click on the **OVA file**. Follow
-the instructions to import the virtual machine.
-Edit your virtual machine and in the **System** tab, check the
-**Hardware Clock in UTC Time** box, then click on **OK**.
 
-![image](../../assets/installation/ova_vbox_use_hardware_clock.png)
+1. Import the **centreon-central.ova** file into VMWare. A terminal window opens: wait for the server to start. When it is ready, the terminal shows the following message:
```
Comment (tuntoja) [inline en/installation/installation-of-a-central-server/using-virtual-machines.md:40]:
```suggestion
1. Import the **centreon-central.ova** file into VMWare.
```

File: fr/installation/installation-of-a-central-server/using-virtual-machines.md [hunk 3]
```diff
@@ -1,106 +1,207 @@
...
-<!--VMware environment-->
-\
-Accédez à **Fichier > Déployer le modèle OVF** et sélectionnez le **fichier OVA**.
-Parce que les sélections de menu sont liées à votre configuration VMWare, nous ne
-pouvons pas fournir plus d'informations. Sachez que la meilleure pratique
-consiste à utiliser l'option **Thin Provision** pour économiser autant d'espace
-libre que possible sur le disque.
+<!--Environnement VMware-->
+
+1. Importez le fichier `centreon-central.ova` dans VMWare. Un terminal s’ouvre : attendez que le serveur démarre. Lorsque celui-ci est prêt, le terminal affiche le message suivant :
```
Comment (tuntoja) [inline fr/installation/installation-of-a-central-server/using-virtual-machines.md:48]:
```suggestion
1. Importez le fichier `centreon-central.ova` dans VMWare. 
```

Comment (tuntoja) [reply fr/installation/installation-of-a-central-server/using-virtual-machines.md:48]: at this point there is no terminal window yet

File: en/installation/installation-of-a-central-server/using-virtual-machines.md (hunk 2 above)
Comment (tuntoja) [reply en/installation/installation-of-a-central-server/using-virtual-machines.md:40]: at this point there is no terminal window yet

**Review by tuntoja — CHANGES_REQUESTED** [review]

**Review by tuntoja — CHANGES_REQUESTED** [review]
to discuss on thursday

**Review by tuntoja — APPROVED** [review]

**Pull request merged**